### ISSUE TITLE
Game Gear & Sega Master System Support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [TASVideos/gens-rerecording#97]
```
This would really complete the Gens R if you can get it to support game gear 
and sega master system
```

Original issue reported on code.google.com by `shatteredlites` on 2 Jun 2012 at 9:29

**Comment by vgturtle127** [comment]
The Sega Game Gear and Sega Master System are way different than the Sega Genesis/Megadrive. There are other emulators which allow this, they are fairly common actually (KEGA Fusion, for example). This would not be a simple task either. We would have to do a lot of work and testing, and we would have to keep up with the Sega Genesis/Megadrive stuff on top of all of the 32X stuff and Sega CD/Mega CD stuff.

We can either look at other emulators and try to do this, but I think it may be outside the scope of this one emulator with this one team, at least at the moment.

If anybody else would like to weigh in, that would be good.

**Comment by shinra358** [comment]
Kega fusion isn't good anymore. Poor controller support.

**Comment by vgturtle127** [comment]
@shinra358 Yeah, it has issues on modern Windows as well. Not sure what can be done other than improve upon all the Gens forks or maybe contribute to Regen/Exodus/BlastEm?

**Comment by shinra358** [comment]
Hadn't even heard of those but before you wrote this, I just went ahead and did a retroarch for the sega stuff because I saw that it had to controller support and the ability to allow me to create my own perfect scanlines as an overlay. So I'll stick to that for now. Either of your solutions sound good. But I would choose improving on Gens forks considering it has great compatibility and that those others seem brand new and are still working towards that goal of game compatibility.
